runtime/src/main/java/org/corfudb/protocols/wireprotocol/StreamsAddressRequest.java

```diff
@@ -0,0 +1,61 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NonNull;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Represents the request sent to the sequencer to retrieve one or several streams address map.
+ *
+ * Created by annym on 02/06/2019
+ */
+@Data
+@AllArgsConstructor
+@CorfuPayload
+public class StreamsAddressRequest implements ICorfuPayload<StreamsAddressRequest>{
+
+    public static final byte STREAMS = 0; /*To request specific streams*/
```

Comment:
Replace all occurrences of single line comments in `/*....*/`  format  with `//` in this patch.